fractions/microprofile/microprofile-fault-tolerance/src/main/resources/modules/org/eclipse/microprofile/faulttolerance/main/module.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ Copyright 2017 Red Hat, Inc, and individual contributors.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<module xmlns="urn:jboss:module:1.3" name="org.eclipse.microprofile.faulttolerance">
+  <resources>
+    <artifact name="org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api:${mpft-api.version}"/>
+    <artifact name="com.netflix.hystrix:hystrix-core:${hystrix.version}"/>
+    <artifact name="org.wildfly.swarm:mp_fault_tolerance_cdi_extension:${project.version}"/>
+    <artifact name="io.reactivex:rxjava:1.2.5" />
```

Comment:
I think that hystrix and rxjava resources should be replaced with `com.netflix.hystrix` module dependency.

Reply:
this custom module seems wrong.

If it's an Eclipse MP FT module then it shouldn't have rxjava, hystrix, or our cdi extension as dependencies.

Is this trying to be a deployment module? I think there's a better way to do that